nix/scripts.nix

```diff
@@ -0,0 +1,55 @@
+{ pkgs
+, config
+, cronos ? (import ../. { inherit pkgs; })
+}:
+rec {
+  start-cronos = pkgs.writeShellScriptBin "start-cronos"
+    ''
+      set -e
+      if [ -z "$1" ]; then
+        echo "No data directory supplied"
+        exit 1
+      fi
+      export PATH=${pkgs.pystarport}/bin:${cronos}/bin:$PATH
+      pystarport serve --config ${config.cronos-config} --data $1
+    '';
+  start-geth = pkgs.writeShellScriptBin "start-geth"
+    ''
+      set -e
+      if [ -z "$1" ]; then
+        echo "No data directory supplied"
+        exit 1
+      fi
+      export PATH=${pkgs.go-ethereum}/bin:$PATH
+      geth --datadir $1 init ${config.geth-genesis}
+      pwdfile=$(mktemp /tmp/password.XXXXXX)
+      tmpfile=$(mktemp /tmp/validator-key.XXXXXX)
+
+      cat > $pwdfile << EOF
+      123456
+      EOF
+
+      # import validator key
+      cat > $tmpfile << EOF
+      826E479F5385C8C32CD96B0C0ACCDB8CC4FA5CACCC1BE54C1E3AA4D676A6EFF5
```

Comment:
from Geth genesis?

Reply:
there's no private key in the genesis file though.

Reply:
I'd like to fix the hardcoded stuff in another PR.
https://github.com/crypto-org-chain/cronos/issues/10